src/doc/en/developer/coding_in_python.rst

```diff
@@ -37,14 +37,34 @@ using one of two mechanisms:
 - Backport packages
 
   - `importlib_metadata <../reference/spkg/importlib_metadata>`_
-    (to be used in place of ``importlib.metadata``),
+    (to be used in place of ``importlib.metadata`` when Python older
```

Comment:
Not related with this PR, but you may want to fix the line 18.

Reply:
I've pushed a change in 52001daab13, not sure if this is what you had in mind

Reply:
> I've pushed a change in [52001da](https://github.com/sagemath/sage/commit/52001daab13c74cfa3291c8270c2140457ff96b9), not sure if this is what you had in mind

No. I thought I saw a sentence mentioning `src/setup.cfg.m4`, which is gone. 

Thanks, anyway :-)

Reply:
Ah, this may have disappeared in the rebasing that I did.